### ISSUE TITLE
Suggestion: zoom level

### ISSUE THREAD

**Issue by Nightfirecat** [dillydill123/inventory-setups#162]
Because I play in fixed mode, I find for a number of bosses and raids it becomes helpful to zoom out further than I would normally. So far, I've noted this by adding a line in the additional notes section to the effect of "Zoom: default -4". (meaning to reset to default zoom, then scroll back 4 times) It would be useful to be able to set this programmatically from the plugin so I wouldn't need to reference the notes every time.

**Comment by dillydill123** [comment]
I'm not sure about setting the zoom level automatically. I think I could add a toggle for it. Or I could make a specific panel that shows if your zoom level is off. There are a lot of small miscellaneous things I was thinking of putting in a Miscellaneous panel like Attack style, quick prayers, pet, zoom level, etc.

**Comment by Nightfirecat** [comment]
I agree it would be a better fit for a miscellaneous-style panel than something that every setup would want
